### PR TITLE
Uncommented or removed commented out code in std.conv

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/conv.d [hunk 1]
```diff
@@ -1061,7 +1059,7 @@ if (is (T == immutable) && isExactSomeString!T && is(S == enum))
     assertCTFEable!dg;
 }
 
-@safe pure /+nothrow+/ unittest
```
Comment (burner) [inline std/conv.d]: have you tested if it is nothrow?

File: std/conv.d [hunk 2]
```diff
@@ -4655,20 +4634,6 @@ unittest
...
-    //{
-    //    struct S3
-    //    {
-    //        int k;
-    //        void foo(){++i;}
-    //    }
-    //}
-    //S3 s3 = void;
-    //emplace(&s3);    //S3.init has no context pointer information
-    //emplace(&s3, 1); //No way to obtain context pointer once inside emplace
```
Comment (burner) [inline std/conv.d]: again, has you tested this

File: std/conv.d (hunk 1 above)
Comment (JackStouffer) [inline std/conv.d]: Yes, many of the tests in the inner loop throw.

File: std/conv.d (hunk 2 above)
Comment (JackStouffer) [inline std/conv.d]: Yes, this no longer compiles as a static assert was added to emplace.

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Pull request merged**